test/app/database/models/numeric-id.js

```diff
@@ -0,0 +1,10 @@
+import mongoose from "mongoose";
+
+const schema = mongoose.Schema({
+  _id: {
+    type: Number,
+    required: true
```

Comment:
> Wouldn't serializedId = String(unserializedId) cover 99% of use cases?

Yup, and that's basically what I'm thinking. That logic just has to be applied (at least in the Document type, but maybe other places I'm not thinking of).